src/lib/iframe-tools/iframe-manager.js

```diff
@@ -21,6 +21,10 @@ export class IframeManager {
   };
 
   _onMessage = (e) => {
+    if (!e || !e.source) {
```

Comment:
sugg: can you add a comment here about the issue that caused you to add the condition ?